internal/server/cache/cache.go

```diff
@@ -0,0 +1,67 @@
+package cache
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/go-redis/redis/v8"
+	rate "github.com/go-redis/redis_rate/v9"
+
+	"github.com/infrahq/infra/internal/logging"
+)
+
+type Cache struct {
+	client *redis.Client
+}
+
```

Comment:
why is something that does rate-limiting called cache?

Reply:
That's a fair point. I was thinking to generalize Redis as "cache" but you're correct that Redis is more than a cache.